docs/book/src/topics/getting-started.md

```diff
@@ -54,6 +54,8 @@ An Azure Service Principal is needed for deploying Azure resources. The below in
 :warning: NOTE: If your password contains single quotes (`'`), make sure to escape them. To escape a single quote, close the quoting before it, insert the single quote, and re-open the quoting.
 For example, if your password is `foo'blah$`, you should do `export AZURE_CLIENT_SECRET='foo'\''blah$'`.
 
+:warning: The capability to set credentials using environment variables is now deprecated and will be removed in future releases, the recommended approach is to use `AzureClusterIdentity` as explained [here](multitenancy.md)
```

Comment:
this can be an aside now that it's part of the book (same for above) 

```
<aside class="note warning">

<h1> Warning </h1>

body of warning

</aside>
```

Reply:
was just looking into why the doc lint failed, will do that